docs/3.0.0-beta.x/plugin-development/plugin-architecture.md

```diff
@@ -3,53 +3,39 @@
 The logic of a plugin is located at its root directory `./plugins/**`. The admin panel related parts of each plugin are contained in the `/admin` folder.
 The folders and files structure are the following:
 
-<!-- ```
-/plugin
-└─── admin // Contains the plugin's front-end
-|     └─── src // Source code directory
-|          └─── index.js // Entry point of the plugin
-|          └─── pluginId.js // Name of the plugin
-|          └─── lifecycles.js // File in which the plugin sets the hooks to be ran in another plugin.
+```
+plugin/
+└─── admin/ # Contains the plugin's front-end
+|     └─── src/ # Source code directory
+|          └─── index.js # Entry point of the plugin
```

Comment:
Can we remove the above structure as it is not up to date with the current architecture regarding the front-end files?

Reply:
Should it be updated instead of deleted?

Reply:
I will update the file with the media library feature so can we delete it for now?

Reply:
What part of the structure @soupette ?

Reply:
If you could remove the everything that is in under the `HomePage` it would be awesome. (L-19-L25)